### PR TITLE
docs(contributing): alternative way to test

### PR DESCRIPTION
**Pull request by ematipico**
- [ ] This is a **bugfix**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **typo fix**
- [X] This is a **metadata update**

### For Bugs and Features; did you add new tests?
N/A

### Motivation / Use-Case
Propose a different way to test the local fork without the need to push to github. I use this way then I want to install deps that live in my local machine

### Breaking Changes
N/A

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -37,13 +37,21 @@ If you are modifying a file in the `client/` directory, be sure to run `npm run
...
 
-1. In your `package.json`, change the line with `webpack-dev-server` to:
+1a. In your `package.json`, change the line with `webpack-dev-server` to:
 
   ```json
   "webpack-dev-server": "github:webpack/webpack-dev-server#pull/<ID>/head"
   ```
 
   `<ID>` is the ID of the pull request.
+OR
```
Comment (michael-ciniawsky) [inline CONTRIBUTING.md]: Can we add docs for [`npm/yarn link`](https://docs.npmjs.com/cli/link) usage here instead?  You can keep `  "webpack-dev-server": "file:../path/to/local/webapck-dev-server/fork"` if you like, but please incoporate it into the first example as an alternative **value**

Comment (ematipico) [reply CONTRIBUTING.md]: I could update the it with `link` but I'd prefer to do it with another PR, if it's ok

**Comment by ematipico** [comment]
PR updated!

**Pull request merged**